packages/ui/src/2_molecules/Pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { Story } from '@storybook/react';
+
+import React, { ComponentProps, useState } from 'react';
+
+import { Pagination } from './Pagination';
+
+export default {
+  title: 'Molecule/Pagination',
+  component: Pagination,
+};
+
+const Template: Story<ComponentProps<typeof Pagination>> = args => {
+  const [page, setPage] = useState(0);
```

Comment:
I don't think that's a good approach to set pagination state from a parent component, do you have a specific use case for this? I think it should have an internal state and in case you need to override it, just set `page` prop.